README.md

```diff
@@ -107,13 +107,13 @@ and/or
 
 | Name | Solidity | Proxy | Implementation | Notes |
 | -------- | -------- | -------- | -------- | -------- | 
-| StrategyManager | [`StrategyManager`](https://github.com/Layr-Labs/eigenlayer-contracts/blob/0139d6213927c0a7812578899ddd3dda58051928/src/contracts/core/StrategyManager.sol) | [`0x779...8E907`](https://goerli.etherscan.io/address/0x779d1b5315df083e3F9E94cB495983500bA8E907) | [`0x3FFa...3dE6`](https://goerli.etherscan.io/address/0x3FFa9daE46d15f15c925d4694c19c49e624b3dE6) | Proxy: [OpenZeppelin TUP@4.7.1](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.7.1/contracts/proxy/transparent/TransparentUpgradeableProxy.sol) |
+| StrategyManager | [`StrategyManager`](https://github.com/Layr-Labs/eigenlayer-contracts/blob/0139d6213927c0a7812578899ddd3dda58051928/src/contracts/core/StrategyManager.sol) | [`0x779...8E907`](https://goerli.etherscan.io/address/0x779d1b5315df083e3F9E94cB495983500bA8E907) | [`0x8676...0055`](https://goerli.etherscan.io/address/0x8676bb5f792ED407a237234Fe422aC6ed3540055) | Proxy: [OpenZeppelin TUP@4.7.1](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.7.1/contracts/proxy/transparent/TransparentUpgradeableProxy.sol) |
```

Comment:
Just noticing that a handful of these addresses use really inconsistent "preview" formats...

Fix now or in a later PR?

i.e. the mainnet contract addresses above all preview 2 leading bytes and 2 ending bytes.

These goerli addresses just sorta... do a random amount of bytes.

Reply:
Later PR is fine, no worries. I'll make an issue

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/issues/331